docs/Course Wiki/MATH Course Offerings/MATH257.md

```diff
@@ -0,0 +1,56 @@
+# MATH 257
+
+MATH257 (Linear Algebra with Computational Applications) is a 3-credit-hour class that is one of the math courses required for both EE and CE majors as part of the ECE core curriculum. It is offered in the fall, spring, and summer.
+
+## Content Covered
+
+- Linear equations, linear systems
+- Matrix operations, LU decomposition
+- Vector spaces, subspaces
+- Basis and dimension
+- Linear transformations
+- Eigenvalues, eigenvectors
+- Inner products and norms
+- Orthogonality, orthogonal projections, linear regression
+- Equilibrium, linear dynamical systems
+- Gram-Schmidt method, QR decomposition
+- Singular value decomposition, pseudo-inverse
+
+MATH257 is the first linear algebra course that covers many fundamental concepts in linear algebra. It starts with linear equations and linear systems, and introducing the concept of vectors and matrices to solve linear systems. The course then delves into matrix operations such as addition and multiplication, and the different types of matrices such as elementary and inverse matrices. All of these concepts are regularly utilized as you proceed onto more difficult topics such as subspaces, eigenvectors, linear transformations, and orthogonal projections, so make sure you are comfortable with the fundamentals.
+
+## Prerequisites
+ - [MATH 221](MATH221.md)
```

Comment:
Remove space between MATH and 221, explain what content is important from MATH221 for MATH257.